src/ape_accounts/accounts.py

```diff
@@ -160,7 +175,10 @@ def _prompt_for_passphrase(self, message: Optional[str] = None, **kwargs):
             **kwargs,
         )
 
-    def __decrypt_keyfile(self, passphrase: str) -> HexBytes:
+    def __decrypt_keyfile(self, passphrase: Optional[str]) -> HexBytes:
+        # if no passphrase loaded throw an exception
+        if not passphrase:
+            raise InvalidPasswordError()
```

Comment:
Don't quite understand why this is now Optional :thinking:

Reply:
If you can check the return type of passphrase in method `unlock`, it's optional[str] maybe that's why python is complaining. But before that it successfully ran on my machine without breaking anything.

Reply:
you may just have to use a different variable name or assert it is not None or something.
let's make it so it is not optional here.

Reply:
Okay I see what's happening here. It will never be `None` because of how `_prompt_for_passphrase` works (which should probably have a return type hint)